### ISSUE TITLE
BQ dataset used in dataflow example doesn't exist

### ISSUE THREAD

**Issue by dylanmpeck** [GoogleCloudPlatform/professional-services#527]
The `python-dataflow-example.example_data` BigQuery dataset used in the cookbook batch-examples of dataflow-examples code is returning a 404 and all operations with code that references it are failing.

Did this dataset change in some way recently?

Link to two files where I know dataset is referenced:
 https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-python-examples/batch-examples/cookbook-examples/pipelines/data_enrichment.py
https://github.com/GoogleCloudPlatform/professional-services/blob/master/examples/dataflow-python-examples/batch-examples/cookbook-examples/pipelines/data_lake_to_mart.py

**Comment by prathapreddy123** [comment]
We recently moved the code around (without any logic changes) to different directory as part of restructuring.  Can you verify if    call is happening to right path.

**Comment by dylanmpeck** [comment]
We are using a forked version of repo that is still in the state of before the directories were restructured. This is for a Qwiklabs lab that uses this code.

I believe the path is right, but here's a picture with error message:
![Job failed](https://user-images.githubusercontent.com/40506467/88985802-2798d700-d286-11ea-985d-a5fc968b4f3c.png)

To add more info, we only started getting reports of this error since yesterday.

**Comment by prathapreddy123** [comment]
Based on error message project python-dataflow-example seems to be missing. Can you cross check if project and corresponding BQ table exists

**Comment by dylanmpeck** [comment]
I don't think so. In BigQuery within that project, I'm unable to locate the table and I'm unable to do simple queries on that project + table. Always getting a "doesn't exist" related error.

Is the table publicly hosted somewhere? Or, is the table supposed to be generated by the example code?

I had thought both examples were using a public table as a 2nd source based on how the table is referenced.

**Comment by eparizzi** [comment]
The data doesn't not exist or is not set as public.

`gsutil ls -r gs://python-dataflow-example/data_files`returns `BucketNotFoundException: 404`

**Comment by tommo245** [comment]
Hi - Also not finding the necessary template, getting this error.

```
File "/path/to/lib/python3.8/site-packages/google/cloud/_http.py", line 438, in api_request
    raise exceptions.from_http_response(response)
google.api_core.exceptions.NotFound: 404 GET https://storage.googleapis.com/storage/v1/b/python-dataflow-example?projection=noAcl&prettyPrint=false: Not Found
```

**Comment by agold-rh** [comment]
@joshmcginley @lcaggio Do you have any input here, or should this be closed as stale?
